lib/module/doom.go

```diff
@@ -0,0 +1,55 @@
+package module
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/dvdmuckle/irkbot/lib/configure"
+	"github.com/dvdmuckle/irkbot/lib/message"
+	"net/http"
+	"os"
+	"strings"
+)
+
+type doomStruct struct {
+	Type string `json:"type"`
+}
+
+const doomHost = "http://localhost:8000/api/player/actions"
+
+func HelpDoom() []string {
+	s := "doom <command> - play doom!"
+	return []string{s}
+}
+
+//Sanitize input
```

Comment:
This comment makes no sense here.

Reply:
This comment still makes no sense here..... I think you had a git fail when you squashed.

Reply:
Hopefully the third time is a charm......